llvm/include/llvm/CodeGen/SlotIndexes.h

```diff
@@ -22,7 +22,7 @@
 #include "llvm/ADT/IntervalMap.h"
 #include "llvm/ADT/PointerIntPair.h"
 #include "llvm/ADT/SmallVector.h"
-#include "llvm/ADT/ilist.h"
+#include "llvm/ADT/simple_ilist.h"
```

Comment:
This file uses ilist<> (which is a typedef for iplist<>).

Reply:
Sorry. I was trying to do IWYU by hand but it is more complicated than I realized. I will abandon this patch.